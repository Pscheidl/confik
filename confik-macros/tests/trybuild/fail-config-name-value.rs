#[derive(confik::Configuration)]
struct Config {
    #[confik = "foo"]
    _param: String,
}

fn main() {}
