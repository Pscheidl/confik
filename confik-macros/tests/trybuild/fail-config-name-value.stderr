error: Name-value arguments are not supported. Use #[confik(...)]
 --> tests/trybuild/fail-config-name-value.rs:3:7
  |
3 |     #[confik = "foo"]
  |       ^^^^^^
